ui/src/views/setting/ConfigurationValue.vue

```diff
@@ -280,6 +295,9 @@ export default {
         name: configrecord.name,
         value: newValue
       }
+      if (this.scopeKey === 'domainid' && !params[this.scopeKey]) {
```

Comment:
do we need the this.scopeKey === 'domainid' check? Can it be 
```suggestion
      if (this.scopeKey && !params[this.scopeKey]) {
```